adder/dagservice.go

```diff
@@ -0,0 +1,119 @@
+package adder
```

Comment:
the `ipfsadder` thing needs a dag service (https://godoc.org/github.com/ipfs/go-ipld-format#DAGService). As blocks are output from the chunker-importer module (the ipfs importer), they are sent to the DAGService. We provide a custom DAG service which forwards the blocks to us, rather than storing them on the datastores (as the regular one does).